src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlNodeList.cs

```diff
@@ -38,7 +38,7 @@ public override int Count
         public int Add(object value)
         {
             if (!(value is XmlNode))
-                throw new ArgumentException(SecurityResources.GetResourceString("Cryptography_Xml_IncorrectObjectType"), "node");
+                throw new ArgumentException(SR.GetResourceString("Cryptography_Xml_IncorrectObjectType"), "node");
```

Comment:
Ah, I wondered about that. There was no strongly-typed resources generator associated with the initially-supplied resx file in the project so I thought perhaps you guys didn't want it done that way. I'll switch to that and then remove all unreferenced strings.